docs/sources/write/shortcodes/index.md

```diff
@@ -541,6 +541,43 @@ Including the original image dimensions as the 'width' and 'height' properties i
 These values are _only_ used for determining the image aspect ratio and don't equate to the final displayed size.
 {{< /admonition >}}
 
+### Center an image example
+
+You can center an image using the `figure` shortcode by adding the following properties:
+
+- Add the `class="w-100p"` property.
+- Add the `link-class="w-fit mx-auto d-flex flex-direction-column"` property.
+- Add the `max-width="WIDTHpx"` property, replacing `WIDTH` with the value you want for your image. The value should be lower than the original image's width.
+- Optionally, you can add the `width` and `height` properties with the original image values, without pixels or percentages. For example, for a 800x600 image, you can add `width="800" height="600"`.
+
+This is an example for centering an image that's 1275x738 pixels:
+
+```markdown
+{{</* figure src="/static/img/docs/grafana-cloud/k8sPods.png" width="1275" height="738" max-width="500px" class="w-100p" link-class="w-fit mx-auto d-flex flex-direction-column" caption="Pod view in Grafana Kubernetes Monitoring" caption-align="center" */>}}
+```
+
+For more details about what each parameter does, refer to the section below.
+
+{{< collapse title="Center image properties" >}}
```

Comment:
@jdbaldry ok, I added the collapsable section here with some of the information Nate shared with me. 🤓

Would you mind taking a look and letting me know if it all makes sense? And I can go in and fix the Vale issues afterwards.